app/views/pages/home.html.erb

```diff
@@ -27,12 +27,10 @@
     <p class="govuk-body">Review and manage your trainee records.</p>
   </div>
 
-  <div class ="govuk-grid-column-one-third">
-    <h2 class="govuk-heading-m">
-      <%= govuk_link_to "Give feedback", Settings.feedback_link_url, { class: "govuk-link govuk-link--no-visited-state" } %>
-    </h2>
-    <p class="govuk-body">This is a new service. Your views will help us improve it.</p>
-  </div>
+  <%= render FeedbackLinkHome::View.new(
```

Comment:
You've created a new `FeedbackLinkHome` component, but it's not actually used anywhere other than right here. Our view components should really encapsulate re-used pieces of code, or components that contain a lot of their own logic (like a filter component).

You could probably just wrap lines 30-35 in an `if` statement like you've done elsewhere and that saves creating two new files.